cmd/network-helper/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	ctlcni "github.com/harvester/harvester/pkg/generated/controllers/k8s.cni.cncf.io"
+	"github.com/urfave/cli"
+	"k8s.io/klog"
+
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/harvester/harvester-network-controller/pkg/controller/manager/nad"
+	"github.com/harvester/harvester-network-controller/pkg/helper"
+	"github.com/harvester/harvester-network-controller/pkg/utils"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "network-helper"
+	app.Usage = "network-helper is to help get the network information through DHCP protocol from the pod within the VLAN network"
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:   "kubeconfig, k",
+			EnvVar: "KUBECONFIG",
+			Value:  "",
+			Usage:  "Kubernetes config files, e.g. $HOME/.kube/config",
+		},
+		cli.StringFlag{
+			Name:   "master, m",
+			EnvVar: "MASTERURL",
+			Value:  "",
+			Usage:  "Kubernetes cluster master URL.",
```

Comment:
Let's name it "server" like the kubeconfig way

Reply:
It's called masterurl in the function `BuildConfigFromFlags` of client-go. It would be better to keep the same. 
``` go
// BuildConfigFromFlags is a helper function that builds configs from a master
// url or a kubeconfig filepath. These are passed in as command line flags for cluster
// components. Warnings should reflect this usage. If neither masterUrl or kubeconfigPath
// are passed in we fallback to inClusterConfig. If inClusterConfig fails, we fallback
// to the default config.
func BuildConfigFromFlags(masterUrl, kubeconfigPath string) (*restclient.Config, error) {
	if kubeconfigPath == "" && masterUrl == "" {
		klog.Warningf("Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.")
		kubeconfig, err := restclient.InClusterConfig()
		if err == nil {
			return kubeconfig, nil
		}
		klog.Warning("error creating inClusterConfig, falling back to default config: ", err)
	}
	return NewNonInteractiveDeferredLoadingClientConfig(
		&ClientConfigLoadingRules{ExplicitPath: kubeconfigPath},
		&ConfigOverrides{ClusterInfo: clientcmdapi.Cluster{Server: masterUrl}}).ClientConfig()
}
```